### DISCUSSION TITLE
Can I safely upgrade to macOS Ventura and still run AMUSE?

### DISCUSSION THREAD

**Discussion by rieder** [amusecode/amuse#898] [FAQ]
macOS Ventura was released late October 2022.
~At the moment, AMUSE is untested on this OS, so upgrading is *not* recommended!~
~We will start testing when Xcode 14.1 is released.~
**Update:** Xcode 14.1 is now released, and after installing this AMUSE can be installed as usual (tested using GCC 12 and OpenMPI).
So AMUSE is not a roadblock if you're thinking of upgrading your OS!

**Comment by mmaclow** [comment]
It just released today!

**Reply by rieder** [reply]
Then testing will start soon :).

**Comment by rieder** [comment]
Happy to report that with Xcode 14.1, AMUSE installs as usual (but only tested with OpenMPI and GCC 12).
So no roadblocks (as far as AMUSE is concerned) for upgrading to macOS Ventura :).

**Reply by sergio0711** [reply]
Hello, 

I have tried to install AMUSE in my laptop. It's a macOS with Ventura and Xcode 14.2.

I could create the environment and install all prerequisites that are mandatory according to documentation. However, I'm getting an error when I try to install amuse or amuse-framework. I've been thinking that it might be due to a missing package, but I checked everything and looks okay.

**The error I'm getting is:** 

```
ERROR: Failed building wheel for amuse-framework
Failed to build amuse-framework
ERROR: Could not build wheels for amuse-framework, which is required to install pyproject.toml-based projects
```

If I go towards the top of the commentaries, it also mentions:

```
error: subprocess-exited-with-error
  
  × Building wheel for amuse-framework (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [160 lines of output]
      running bdist_wheel
      running build
      /private/var/folders/p4/ghnf3dfn1jg5fytn2f6vbdl40000gn/T/pip-build-env-9cwoierv/overlay/lib/python3.9/site-packages/setuptools/command/build.py:31: SetuptoolsDeprecationWarning:
                  It seems that you are using `distutils.command.build` to add
                  new subcommands. Using `distutils` directly is considered deprecated,
                  please use `setuptools.command.build`.
```

Am I missing something? I don't understand the part which mentions the library _setuptools_. I'm using Python 3.9 and openMPI.


Thanks for your help in advance.


Sergio
